static/docs/commands-reference/pull.md

```diff
@@ -59,8 +122,84 @@ Pull all files from the current Git branch:
     (8/8): [#################################] 100% model.pkl
 ```
 
-Pull outputs of a specific dvc file:
+We can download specific files that are outputs of a specific dvc file:
+
 ```dvc
     $ dvc pull data.zip.dvc
     [#################################] 100% data.zip
 ```
+
+In this case we left off the `--remote` option, so it will have pulled from the
```

Comment:
This is not true anymore?

Reply:
This example shows how leaving off a `--remote` option it uses the remote defined in the config.

Reply:
My bad. 👍